content/help/ops/controlz/index.md

```diff
@@ -22,3 +22,13 @@ Here's sample of the ControlZ interface:
 
 The `--ctrlz_port` and `--ctrlz_address` command-line options can be given when starting a component to control the
 specific address and port where ControlZ should be exposed.
+
+To access the ctrlz page of deployed components (i.e. Mixer, Galley, Pilot), you can port-forward their ControlZ endpoints
+locally and connect through your local browser:
+
+```shell
```

Comment:
Please use:

{{< text bash >}}
...
{{< /text >}}

And add a $ in front of the command-line.